wiki/Installing-Oppia-Android.md

```diff
@@ -63,85 +341,183 @@ Please follow these steps to set up Oppia Android on your local machine.
+
+   ![Image](https://github.com/user-attachments/assets/9baf6f71-560b-49fe-84a7-c7b846a64287)
+
+4. On **OK**, you will be asked to **Trust project**. Accept because we trust the project.
+
+   <img width="1029" alt="Image" src="https://github.com/user-attachments/assets/bfc91f6d-d809-418d-ab20-89d899f12f72" />
+
+5. On **next**, to Select Project View, **select Create from scratch**.
+
+   <img width="1029" alt="Image" src="https://github.com/user-attachments/assets/cdc110d6-2926-4f96-b7d8-117e2c716f76" />
+
+6. A file similar to the one below will be generated:
+
+   <img width="760" alt="Image" src="https://github.com/user-attachments/assets/5a26210f-865d-4048-b5a5-73379984bfb5" />
+
+7. In the above file, modify it so that it looks like below(copy and paste):
+   
+   ```
+   directories:
+      # Add the directories you want added as source here
+      # By default, we've added your entire workspace ('.')
+      .
+      
+   # Automatically includes all relevant targets under the 'directories' above
+   derive_targets_from_directories: false
+   
+   targets:
+   # If source code isn't resolving, add additional targets that compile it here
+      //:oppia_dev_binary
+      //app
+      //app/src/main/...
+      //domain/...
+      //model/...
+      //testing/...
+      //utility/...
+      //data/...
+      //scripts/...
+      
+   additional_languages:
+      # Uncomment any additional languages you want supported
+      # c
+      # dart
+      # kotlin
+      java
+      kotlin
+
+   # Please uncomment an android-SDK platform. Available SDKs are:
+      # android_sdk_platform: android-28
+      android_sdk_platform: android-29
+      # android_sdk_platform: android-30
+      # android_sdk_platform: android-31
+      # android_sdk_platform: android-32
+      # android_sdk_platform: android-33
+      android_sdk_platform: android-34
+   
+   shard_sync: true
+   sync_flags:
+      --local_ram_resources=HOST_RAM*.5
+      --discard_analysis_cache
+   
+   java_language_level: 11
+   ```
+   **Note**: Only enable Android-SDK platforms that you have installed per the [Prepare developer environment](#prepare-developer-environment) section above. Android-SDK platforms are useful for creating emulators to test your app with.
+
+8. Click **Create**, and allow the project to synchronize.
+
+9. Once sync has finished, you can now build and install the app on either a virtual or physical device. Bazel supports deploying to only one device at a time, so you can connect one device, or launch one emulator at a time.
+
+   You can run the project by using the Bazel plugin to set up run configurations for the target that you wish to build. This performs the same action as the commands below, but using the GUI to run the app might be more intuitive for some developers.
```

Comment:
FWIW we can actually save these configurations for people to use, too:

https://ij.bazel.build/docs/run-configurations.html#sharing-run-configurations

Perhaps that's worth doing and updating the project view above to import it?

Reply:
I have added an action item for this in https://github.com/oppia/oppia-android/issues/1723